### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
What is missing: 
(1) Registering this documentation for rendering and publishing at docs.typo3.org with the [webhook](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocForExtension/Webhook.html#github).
(2) Adding the new manual URL `https://docs.typo3.org/p/friendsoftypo3/form-legacy/main/en-us/` to the TER extension page configuration.

**Comment by alexander-nitsche** [comment]
How it looks:
Screen Shot 2022-04-03 at 12 38 02

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

In addition, the TER homepage is missing the "Code Insights", "Found an issue?" and "Packagist" buttons, which seem to be caused by the empty "Link to repository", "Link to issue tracker", "Composer name of extension" fields in the TER extension edit form.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Contribution/Index.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Feel free to submit any pull request, or add documentation, tests, as you please.
+We will publish a new version every once in a while, depending on the amount of
+changes and pull requests submitted.
+
+Authors
+-------
+
+Over 100 contributors have been working on this area while this functionality
+was part of the TYPO3 Core. This package is now maintained by a loose group of
+TYPO3 enthusiasts inside the TYPO3 Community. Feel free to contact `Benni Mack`_
```
Comment (ohader) [inline Documentation/Contribution/Index.rst]: Please avoid pointing to Benni here, refer to the issue tracker instead...
(seems that basically all legacy extensions had that, I wonder why...)

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid author pointing

File: Documentation/Contribution/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Contribution/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
TER foo done

**Pull request merged**